app/routes.py

```diff
@@ -1 +1,126 @@
-from flask import Blueprint
\ No newline at end of file
+from os import abort
+from app import db
+from app.models.task import Task
+from flask import Blueprint, jsonify, abort, make_response, request 
+import requests
+from datetime import datetime
+import os
+from dotenv import load_dotenv
+
+tasks_bp=Blueprint('tasks_bp',__name__,url_prefix='/tasks')
+
+def validate_task(task_id):
+    try:
+        task_id = int(task_id)
+    except:
+        abort(make_response({"details":f"Invalid data"}, 400))
+
+    
+    task = Task.query.get(task_id)
+
+    if not task:
+        abort(make_response({"message":f"task {task_id} not found"}, 404))
+
+    return task
+
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    # handles if title or description not included in post
+    if "title" not in request_body or "description" not in request_body:
+        return make_response({"details":"Invalid data"},400)
+
+    new_task = Task(title=request_body["title"],
+                    description=request_body["description"])
+
+    db.session.add(new_task)
+    db.session.commit()
+
+    task_response = {"task": new_task.to_dict()}
+
+
+    return make_response(jsonify(task_response), 201)
```

Comment:
We can also omit `make_response` and `jsonify` if we're returning a dictionary. This comment applies to the rest of functions using `jsonify` on dictionaries.

Typically, we can use `jsonify` if we're returning non-dict data types like lists, strings, etc.